extensions/azurehybridtoolkit/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure SQL Hybrid Cloud Toolkit *(preview)*
 
-Adds a Jupyter Book that has several utilities for Azure SQL Hybrid Cloud.
+Adds a Jupyter Book that has several utilities for Azure SQL Hybrid Cloud. After installing the extension, open the Command Palette from the View menu typing a portion of the name of this toolkit. The Notebook should show in the Provided Notebooks panel of the ADS Notebooks Explorer.
```

Comment:
I think the instructions are a little unclear for someone not familiar with ADS, might I suggest this instead : 


Adds a Jupyter Book that has several utilities for Azure SQL Hybrid Cloud. To open the Book install the extension and then
* Open the Command Palette (`View` menu -> `Command Palette...`)
* Type `Jupyter Books: Azure SQL Hybrid Cloud Toolkit` and select that command
* Press Enter

The book should then open in the `Provided Books` panel in the `Notebooks` viewlet